doc/rst/source/explain_-V_full.rst_

```diff
@@ -1,11 +1,12 @@
 .. _-V_full:
 
 **-V**\ [*level*]
-    Select verbose mode, which will send progress reports to *stderr*.
-    Choose among 6 levels of verbosity; each level adds more messages:
-    **q** - Complete silence, not even fatal error messages are produced.
-    **n** - Normal verbosity: produce only fatal error messages.
-    **c** - Produce also compatibility warnings (same as when **-V** is omitted).
-    **v** - Produce also warnings and progress messages (same as **-V** only).
-    **l** - Produce also detailed progress messages.
-    **d** - Produce also debugging messages.
+    Select verbose mode, which modulates the messages written to *stderr*.
+    Choose among 7 levels of verbosity; each level adds more messages:
+    **q** - Quiet, not even fatal error messages are produced.
+    **e** - Error messages only.
+    **w** - Warnings [Default, same as **-V** only].
```

Comment:
As mentioned in #2346, the default verbose mode is **w**, and **-V** only should be equivalent to **-Vi**.

Reply:
OK, fixed and updated in source as well.